### PR TITLE
Clean up org.eclipse.kura.core.log

### PR DESCRIPTION
**Pull request by ctron**
* Convert to Gogo shell
 * Several minor cleanups

Signed-off-by: Jens Reimann <jreimann@redhat.com>

### CODE REVIEW

File: kura/org.eclipse.kura.core.log/pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2011, 2014 Eurotech and/or its affiliates
+    Copyright (c) 2011, 2014 Eurotech and others
```
Comment (amitjoy) [inline kura/org.eclipse.kura.core.log/pom.xml]: 2011, 2016 Eurotech, and/or its affiliates and others

File: kura/org.eclipse.kura.core.log/src/main/java/org/eclipse/kura/core/log/LoggerCommandProvider.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2011, 2016 Eurotech and/or its affiliates
+ * Copyright (c) 2011, 2016 Eurotech and others
```
Comment (amitjoy) [inline kura/org.eclipse.kura.core.log/src/main/java/org/eclipse/kura/core/log/LoggerCommandProvider.java]: 2011, 2016 Eurotech, and/or its affiliates and others

File: kura/org.eclipse.kura.core.log/pom.xml (hunk 1 above)
Comment (ctron) [reply kura/org.eclipse.kura.core.log/pom.xml]: @dwoodard1 Could you confirm which identifier you want as legal entity for Eurotech: "Eurotech" or "Eurotech and/or its affiliates".

Comment (dwoodard1) [reply kura/org.eclipse.kura.core.log/pom.xml]: "Eurotech and/or its affiliates"

**Comment by ctron** [comment]
Done

**Comment by ctron** [comment]
Rebased. Could you please review.

**Comment by dwoodard1** [comment]
@ctron Can you update the Copyright info to 2017? If so, I think this is ready to go.

**Comment by ctron** [comment]
@dwoodard1 Can I directly merge it aftwards?

**Pull request merged**